lib/talis-chart/talis-region.ts

```diff
@@ -0,0 +1,6 @@
+export enum TalisRegion {
+    CANADA = "ca-central-1",
+    EU = "eu-west-1",
```

Comment:
Here the regions are `eu` and `ca`, these are AWS regions - I think that should be renamed in CDK library (a breaking change)